### ISSUE TITLE
Update gone wrong 2.0.1 - 2.0.5

### ISSUE THREAD

**Issue by philippstakenborg** [campuspress/divi-accessibility#78]
Hello,

thanks for all your work with this beautiful plugin! 

I just updated a client site and get some strange errors above my footer. I deactivated the plugin and everything is fine. 
I think I am gonna try older versions of the plugin, 2.0.1 works without problems for me. But I wonder how to get the newest version without the error. Can somebody look into it? 

Warning: Undefined array key "hide_aria_element" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 408

Warning: Undefined array key "show_for_screen_readers_only" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 411

Warning: Undefined array key "hide_aria_element" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 408

Warning: Undefined array key "show_for_screen_readers_only" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 411

Warning: Undefined array key "hide_aria_element" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 408

Warning: Undefined array key "show_for_screen_readers_only" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 411

Warning: Undefined array key "hide_aria_element" in wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 408

Warning: Undefined array key "show_for_screen_readers_only" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 411

Warning: Undefined array key "hide_aria_element" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 408

Warning: Undefined array key "show_for_screen_readers_only" in /wp-content/plugins/divi-accessibility-master/public/class-divi-accessibility-public.php on line 411

Thanks,
Philipp

**Comment by BeginHere** [comment]
What version of PHP is the site using? Did you disable all other plugins and still get errors?

**Comment by philippstakenborg** [comment]
I am using PHP 8.1. - no I just disabled divi accesibility and the error was gone. 
I can install again and see if it is is corelating with other plugins.

**Comment by gjeflec** [comment]
I manage about 100 websites all running Divi and Divi Accessibility under PHP 8.1. No problems like that with any of them so is it possible there's some plugin conflict or some server setting that might be at fault?

**Comment by guardiandesigns** [comment]
I'm getting the same errors on my end.

**Comment by maniu** [comment]
Hey, its definitely something to fix but it sounds like your WP installation is configured to display PHP warnings. Please look into disabling that.

**Comment by guardiandesigns** [comment]
Yes that's enabled with Bulletproof Security, which is where we are getting the warnings. We can disable it no problem.

**Comment by RichardGeorgeDavis** [comment]
Final 2.1.0 follow-up:

The PHP warning path reported here for missing `hide_aria_element` / `show_for_screen_readers_only` module attributes is addressed in #121 and included in the final maintained-fork 2.1.0 release. The module accessibility values are now read through guarded D4/D5 prop access, so missing legacy attributes should not produce undefined-array-key warnings.

Maintained fork: https://github.com/RichardGeorgeDavis/divi-accessibility
Release: https://github.com/RichardGeorgeDavis/divi-accessibility/releases/tag/2.1.0
Zip SHA-256: `69c25e3bbda5d033dacda63ae4814c263c2afa4583be5ac5635caef36109faef`

I do not have permission to close this upstream issue. Maintainers: please close this as fixed by #121 after review if the original warning is no longer reproducible.
